.travis.yml

```diff
@@ -46,7 +43,7 @@ matrix:
           env: CONFIG_OPTS="--debug --coverage no-asm enable-rc5 enable-md2 enable-ec_nistp_64_gcc_128 enable-ssl3 enable-ssl3-method enable-nextprotoneg enable-weak-ssl-ciphers enable-external-tests no-shared -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" COVERALLS="yes" BORINGSSL_TESTS="yes" CXX="g++-5"
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
```

Comment:
This is not really related question, but why does msan enforces no-asm through Configure, but not any other `*san`? Using sanitizers with no-asm does make [a lot of] sense, but is it so that msan is per definition useless without no-asm?